runtime/src/main/java/com/flipkart/flux/guice/module/ExecutionTaskModule.java

```diff
@@ -17,9 +16,10 @@ public ExecutionTaskModule() {
 
     @Override
     protected void configure() {
+       // bind(AuthTokenService.class).toProvider(AuthTokenServiceProvider.class).in(Singleton.class);
+//        bind(FluxRuntimeConnector.class).to(FluxRuntimeConnectorHttpImpl.class).in(Singleton.class);
         bind(RouterRegistry.class).to(EagerInitRouterRegistryImpl.class).in(Singleton.class);
         bind(ExecutableRegistry.class).annotatedWith(ManagedEnv.class).to(TaskExecutableRegistryImpl.class);
         requestStaticInjection(AkkaTask.class);
-        install(new FluxClientComponentModule());
```

Comment:
why is this getting deleted?

Reply:
This module is directly supplied to the injector in the main bootstrap class, FluxInitializer. This is done to keep the modules supplied to injector similar to orchestration and execution roles.